backend/dataall/modules/worksheets/api/types.py

```diff
@@ -87,14 +87,16 @@
     fields=[
         gql.Field(name='worksheetQueryResultUri', type=gql.ID),
         gql.Field(name='queryType', type=gql.NonNullableType(gql.String)),
-        gql.Field(name='sqlBody', type=gql.NonNullableType(gql.String)),
+        gql.Field(name='sqlBody', type=gql.String),
         gql.Field(name='AthenaQueryId', type=gql.NonNullableType(gql.String)),
         gql.Field(name='region', type=gql.NonNullableType(gql.String)),
         gql.Field(name='AwsAccountId', type=gql.NonNullableType(gql.String)),
-        gql.Field(name='AthenaOutputBucketName', type=gql.NonNullableType(gql.String)),
-        gql.Field(name='AthenaOutputKey', type=gql.NonNullableType(gql.String)),
-        gql.Field(name='timeElapsedInSecond', type=gql.NonNullableType(gql.Integer)),
+        gql.Field(name='ElapsedTimeInMs', type=gql.Integer),
```

Comment:
super nit: except names (AWS, Athena) name fields with initial letter in lowercase

Reply:
changes are done